core/modules/taxonomy/taxonomy.module

```diff
@@ -1669,9 +1726,30 @@ function taxonomy_field_settings_form($field, $instance, $has_data) {
     );
   }
 
+  if (module_exists('language')) {
+    $form['follow_interface_language'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Follow interface language'),
+      '#default_value' => $field['settings']['follow_interface_language'],
+      '#description' => t('If selected vocabulary terms will be autocompleted in the language of the website interface.'),
```

Comment:
I will update the wording of this to be closer to what you suggest @klonos. But just wanted to mention that we shouldn't put a comma after "If enabled" since it is a required clause.

Reply:
Not entirely sure what you mean @herbdool (please keep in mind that English is not my mother tongue), but AFAIK, one needs to:

> Use a comma after the if-clause when the if-clause precedes the main clause.

Source: https://www.grammarly.com/blog/conditional-sentences